kie-server-parent/kie-server-services/kie-server-services-prometheus/src/main/java/org/kie/server/services/prometheus/PrometheusKieServerExtension.java

```diff
@@ -129,7 +152,51 @@ public void init(KieServerImpl kieServer, KieServerRegistry registry) {
         }
 
         initialized = true;
-        LOGGER.info("{} started", toString());
+        if (!customMetrics.hasCustomMetrics()) {
+            LOGGER.info("{} started", toString());
+        } else {
+            LOGGER.info( toString() + " started with custom Prometheus metrics provider(s): " + customMetrics.customMetricsProviders());
```

Comment:
instead of concatenating the string it is a better practice to use _{}_ and add the values as parameters.